src/test/java/test/utils/domain/NoIdEntity.java

```diff
@@ -0,0 +1,34 @@
+package test.utils.domain;
+
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.keyvalue.annotation.KeySpace;
+
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * An entity with no {@link Id} annotation
+ *
+ * @author Gokhan Oner
+ */
+@KeySpace
+public class NoIdEntity implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    private String name;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    @Override
+    public String toString() {
+        return "NoIdEntity [name=" + name + "]";
```

Comment:
If you need to have this method (see the previous comment), then use `String.format()`.